src/v3/src/components/Widget/index.tsx

```diff
@@ -291,7 +291,6 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
               />
               <AuthContent>
                 <IdentifierContainer />
-                <InfoSection message={message} />
                 {
                   uischema.elements.length > 0
                     ? <Form uischema={uischema as UISchemaLayout} />
```

Comment:
If `<InfoSection>` moved inside the `<Form>`, don't you have to pass down the prop?